superset-frontend/src/components/Chart/ChartRenderer.jsx

```diff
@@ -172,6 +196,22 @@ class ChartRenderer extends React.Component {
     }
   }
 
+  handleOnContextMenu(filters, offsetX, offsetY) {
+    this.contextMenuRef.current.open(filters, offsetX, offsetY);
+    this.setState({ inContextMenu: true });
+  }
+
+  handleContextMenuSelected(filters) {
+    const extraFilters = this.props.formData.extra_form_data.filters || [];
```

Comment:
I'm getting an error on this line when I try to select "Drill by x": `Cannot read properties of undefined (reading 'filters')`.  I think it might be because I'm running it with `DRILL_TO_DETAIL` only and not `DASHBOARD_CROSS_FILTERS`, and it seems like the `extra_form_filters` isn't initialized as an empty object when `DASHBOARD_CROSS_FILTERS` isn't enabled.

Reply:
Added a safe-guard 👍🏼